content/en/docs/faas/lambda-auto-instrument.md

```diff
@@ -56,7 +55,6 @@ For example, to only enable auto-instrumentation for Lambda and the AWS SDK, you
 would have to set the following environment variables:
 
     ```bash
```

Comment:
Unindent this code block.

Reply:
Also the list items at lines 51 and 52 above need to be unindented. 

Would you mind fixing them? Thanks!

Reply:
Thanks!